CHANGELOG.md

```diff
@@ -1,19 +1,62 @@
-# 2.1.0 (2017-01-XX)
+# 2.1.0 (2017-02-TBD)
+### Version highlights:
+1. Added full support to OneView Rest API version 300 for the hardware variants C7000 and Synergy to the already existing features.
+2. Implemented a Oneview_resource class for the resources to inherit from, reducing code duplication and complexity.
```

Comment:
> Implemented a Oneview_resource class for the resources to inherit from [...]

I guess it could sound better if it was:
*Implemented the Oneview_resource class to act as parent for the other resources [...]*